.github/ISSUE_TEMPLATE/grant_completion.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
I am not sure we will need that anymore because IMO it's better if all of the grant lifecycle happens on the same Github issue. If we have a special template for completion, this would likely open another issue which is not what we'd want. 

That said I think it would be good to have a way to ask the user to complete that as part of the initial grant application thread/issue?